### PR TITLE
use case update in appstatus api

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: cx-portal/src/components/shared/basic/ReleaseProcess/ValidateAndPublish/index.tsx [hunk 1]
```diff
@@ -199,7 +199,10 @@ export default function ValidateAndPublish({
   const getAppData = (item: string) => {
     if (item === 'language')
       return statusData?.supportedLanguageCodes.join(', ')
-    else if (item === 'useCase') return statusData?.useCase.join(', ')
+    else if (item === 'useCase')
+      return statusData?.useCase
+        ?.map((item: { id: string; label: string }) => item.label)
```
Comment (manojava-gk) [inline cx-portal/src/components/shared/basic/ReleaseProcess/ValidateAndPublish/index.tsx:204]: Can you create a separate type/interface for this? This has been used in multiple places

File: cx-portal/src/features/appManagement/types.ts [hunk 2]
```diff
@@ -36,7 +36,10 @@ export interface AppStatusDataState {
   provider?: string
   leadPictureId?: string
   providerName?: string
-  useCase?: string[]
+  useCase?: {
+    id: string
+    label: string
```
Comment (manojava-gk) [inline cx-portal/src/features/appManagement/types.ts:41]: another place

File: cx-portal/src/components/shared/basic/ReleaseProcess/ValidateAndPublish/index.tsx (hunk 1 above)
Comment (lavanya-bmw) [reply cx-portal/src/components/shared/basic/ReleaseProcess/ValidateAndPublish/index.tsx]: done.

**Review by manojava-gk — APPROVED** [review]
Looks okay

**Pull request merged**